dev-requirements.txt

```diff
@@ -0,0 +1,4 @@
+--find-links https://github.com/mracter/hvac/archive/b0811f0446760125baefea6383711035104e6df1.zip#egg=hvac-0.2.17+git.b817da4
+hvac==0.2.17+git.b817da4
```

Comment:
I think rather use `git+https://github.com/mracter/hvac/archive/b0811f0446760125baefea6383711035104e6df1.zip#egg=hvac-0.2.17+git.b817da4` to explicitly specify the bits to install instead of using `--find-links` and a magic version number.